Assets/Scripts/Game/Guilds/DarkBrotherhood.cs

```diff
@@ -79,6 +80,16 @@ public class DarkBrotherhood : Guild
 
         #region Guild Membership and Faction
 
+        public DarkBrotherhood()
+        {
+            RegisterEvents();
```

Comment:
This is not needed since events are registered when joining the guild or when loading the game.